jupyterlab_bigquery/src/utils/pagedAPI/cache_manager.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+class CacheManager {
```

Comment:
Will add implementation in future PR.  Added as placehold for structure consistency.

Reply:
Decided not to implement cache.  Will delete in a future PR.

Reply:
In merging with master, I deleted cachemanager along the way.